test/unit/postal-code.js

```diff
@@ -57,4 +57,21 @@ describe('postalCode', function () {
       });
     });
   });
+
+  describe('custom min length', function () {
+    it('allows passing in a custom min length', function () {
+      expect(postalCode('123')).to.deep.equal({
```

Comment:
Is this testing that the second param is optional? If so I think this case is already covered in the other tests and can be removed